docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -857,6 +857,99 @@
       </sect3>
     </sect2>
 
+    <sect2 id="Egress-HYAML" role="brk">
+      <title>Egress</title>
+
+      <para>In order to allow clusters to be securely locked down but still
+      allow access to the services they need. We have a similar mechanism to
+      Ingress that specifies what external endpoints it is permitted to
+      connect to egress. Keep in mind that Thor components do have their own
+      network policies.</para>
+
+      <para>For instance, in a default deployed system with network policies
+      enforced, a query running (on hThor, Thor, or Roxie), will not be able
+      to connect with a 3rd party service, such as an LDAP service or log
+      stack. Currently, allows any pod that happens to have been given Kube
+      API access, that doesn't specifically have its own pod selected
+      <emphasis>NetworkPolicy</emphasis>, to access any external endpoint.
+      Deploying generates a "default-deny-all-but-dns" which prevents all but
+      DNS, however, we also generate "allow-apiserver" (also with no
+      pod-selector), which by default is:</para>
+
+      <para><programlisting>kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-apiserver
+spec:
+  policyTypes:
+  - Egress
+  podSelector: {}
+  egress:
+  - to:
+    ports:
+    - protocol: TCP</programlisting></para>
+
+      <para>We have added a mechanism similar to the visibilities definitions,
+      which allows named egress sections, which can then be referenced per
+      component.</para>
+
+      <para>For example:</para>
+
+      <programlisting>global:
+  egress:
+    engineEgress:
+    - to:
+        - ipBlock:
+            cidr: 201.13.21.0/24
+        - ipBlock:
+            cidr: 142.250.187.0/24
+      ports:
+        - protocol: TCP
+          port: 6789
+        - protocol: TCP
+          port: 7890
+...
+
+thor:
+...
+  egress: engineEgress</programlisting>
+
+      <para>The Egress section in the default stock/delivered HPCC Systems
+      YAML file, offers more information:</para>
+
+      <programlisting>  egress:
+    ## If restricted is set, NetworkPolicies will include egress restrictions to allow connect
```

Comment:
this text is a truncated version of the values.yaml text, e.g. values.yaml text says "... to allow connections from pods only to the minimum required by the system"

NB: same on other sentences below.

I think it would be better to provide a link to the stock values file rather than copy & paste it here.
e.g.:
https://github.com/hpcc-systems/helm-chart/blob/d2f1239513abab32bcc559fd7f19847dd09795c2/helm/hpcc/values.yaml#L43

Reply:
yes I manually did that to "fit on a printed page" - agreed it should be referred to and not printed out